docs/contributor/benchmarks/otlp-logs-validation.md

```diff
@@ -0,0 +1,407 @@
+    - Agent RECEIVED/EXPORTED: 22.4K / instance
+  - **Results (batching + filestorage):**
+    - Agent RECEIVED/EXPORTED: 20K / instance
+- 📥 OTEL Exporter:
+  - **Results (batching + filestorage):**
+    - Agent RECEIVED/EXPORTED: 15K / instance
+  - **Results (batching + filestorage + sending queue):**
+    - Agent RECEIVED/EXPORTED: 15K / instance
+
+#### 🪲 17 Jan 2025, ~13:16
+- No gateway involved, agent sending directly to mock backend
+- With Istio
+- **2 nodes:**
+  - **Generator:** 60 replicas (10m CPU limit)
+  - Agent RECEIVED/EXPORTED: 28.6K
+  - Agent Memory: 78/71
+  - Agent CPU: 1.3/1.3
+- **3 nodes:**
+  - **Generator:** 90 replicas (10m CPU limit)
+  - Agent RECEIVED/EXPORTED: 44.6K
+  - Agent Memory: ~76-90
+  - Agent CPU: ~1.3
+
+
+## Comparison with FluentBit Setup
+In the FluentBit setup, for the very same (initial) scenario (i.e. 10 generator replicas [old set-up] / 2 agents), the [load test](https://github.com/kyma-project/telemetry-manager/actions/runs/12691802471) outputs the following values for the agent:
+- Exported Log Records/second: 27.8K
+
+
+## Conclusions
+- Before 15 Jan. (first session):
+  - Compared to the FluentBit counterpart setup, a lower performance can be expected.
+  - Backpressure is currently not backpropagated from the gateway to the agent, resulting in logs being queued/lost on the gateway end. That's because the agent has no way of knowing when to stop, thus exports data continuously (this is a known issue, which is expected be solved by the OTel community in the next half year).
+  - If the load is increased (that is, more generators, more logs, or more data), the log agent slows down.
+  - The network communication between the agent and the gateway or/and the gateway represent a bottleneck in this setup. That's concluded because higher throughput was observed when using just a debug endpoint as an exporter.
+  - CPU and memory consumption are surprisingly low, and this was not improved by removing the limits (quite the opposite was observed, with the CPU throttling more often and the throughput decreasing).
+  - If the batch processor is enabled, throughput increased. But this comes at the cost of losing logs in some scenarios.
+  - Further methods of improving the throughput might still be worth investigating.
+- After 15 jan. (second session):
+  - Removing the gateway improves throughput
+  - We now better understand the performance impact of each OTEL processor and of enabling/disabling compression
+  - Generators configuration greatly influence the setup => more generators exporting less data and taking less CPU leads to higher throughput than fewer generators taking more CPU and exporting more data
+  - There is a hard limit (see debug endpoint scenario) that we still not fully understand, since strictly based on the benchmarking numbers of OTEL, we should be getting higher throughput (i.e. something related to the infrastructure could be influencing this).
```

Comment:
```suggestion
  - There is a hard limit (see debug endpoint scenario) that we still don't fully understand, because strictly based on the benchmarking numbers of OTel, we should be getting higher throughput. It's possible that something related to the infrastructure could be influencing this.
```